### PR TITLE
Edit/cypress port

### PR DESCRIPTION
**Pull request by andyforrest**
### Summary of Changes

Added db_port to cypress.config.dist.js to avoid any potential issues with cy.task('queryDB')

### CODE REVIEW

**Comment by richard67** [comment]
@andyforrest Can it be that you have chosen the wrong base branch for this PR? GitHub shows a lot of differences. Maybe you anted to make this PR not for the 4.3-dev branch but for 4.4-dev or 5.0-dev? If that is the case you can switch the base branch by using the edit button beside the title of this pull request on GitHub. Let me know when you need help with that. Thanks in advance.

**Comment by richard67** [comment]
@andyforrest I see you just fixed it, all good. Thanks.

File: tests/System/plugins/db.js [hunk 1]
```diff
@@ -72,7 +72,9 @@ function queryTestDB(joomlaQuery, config) {
   return new Promise((resolve, reject) => {
     // Create the connection and connect
     const connection = mysql.createConnection({
+      // host: config.env.db_host,
```
Comment (richard67) [inline tests/System/plugins/db.js]:
```suggestion
```
Please remove commented out code.

**Comment by laoneo** [comment]
Thanks, can you add the port argument also to the postgres connection, we are using this library https://github.com/porsager/postgres#connection.

**Pull request merged**